.github/workflows/npm_publish.yml

```diff
@@ -24,9 +24,9 @@ jobs:
 
       - name: Publish npm package to both places
         run: |
-          yarn publish --access public
+          yarn publish --access public --tag beta
```

Comment:
since the tag has no ties to versions, it's hard to know what beta tag resolves to, looks like others [have asked about the ambiguity](https://github.com/yarnpkg/berry/issues/2444), do you think it's worth fully qualifying the tag with version, i.e. something like`13.0.0-beta`?

Reply:
Great point. We did `beta` in the past (and of course `beta` just means "latest but not ready") but we have a chance to redefine how we use tags now since it's a new package name. I do think `beta` is fine because it just means "latest but not ready for prod" but we can be clearer. 

What about `protocol-22-beta`? Even less ambiguous.